crates/bindings-csharp/BSATN.Runtime/BSATN/I128.cs

```diff
@@ -0,0 +1,135 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace SpacetimeDB;
+
+using System.Diagnostics.CodeAnalysis;
+using System.Runtime.InteropServices;
+
+/// <summary>Represents a 128-bit signed integer.</summary>
+[StructLayout(LayoutKind.Sequential)]
+public readonly struct I128 : IEquatable<I128>, IComparable, IComparable<I128>
+{
+    internal const int Size = 16;
+
+#if BIGENDIAN
+        private readonly ulong _upper;
```

Comment:
Nit: broken formatting, probably after changing the namespace.

Unfortunately https://github.com/clockworklabs/SpacetimeDB/pull/1567 is still waiting for review, but can you meanwhile install CSharpier and run `dotnet csharpier .` in `crates/bindings-csharp`? That way we have less code to fix once CI check lands.